### ISSUE TITLE
lane coordinate ground truth annotation 

### ISSUE THREAD

**Issue by NcJie** [XingangPan/SCNN#21]
I find that there are some issues with the lane coordinate ground truth annotation. 

1. Part of the `*.lines.txt` are empty 
    - such as `driver_23_30frame/05171102_0766.MP4` has empty `*.lines.txt` files 

2. Number of available lanes listed in the `train_gt.txt` file does not match the corresponding `*.lines.txt` lane 
    - for example `/driver_182_30frame/06011137_0163.MP4/05370.lines.txt` listed 4 lanes coordinates but the `train_gt.txt` file shows `0 1 1 0` indicates that there should be only 2 lanes available   

3. I also try to overlay the `*.lines.txt` points onto the target corresponding image but it doesn't seem to be annotated correctly. The coordinate is in (x, y) form I assume. 

I'm sure there's something I missed. A more detailed description about the lane coordinate ground truth annotation is appreciated. Thank you!

**Comment by XingangPan** [comment]
Thanks for your notice. There's something wrong with the *.lines.txt files. 
Sorry about this. Will fix this issue next Monday.

**Comment by XingangPan** [comment]
Hi NcJie. The annotation files for train and validation set (driver_23, driver_161, driver_182) have been updated. For test set, the annotation files should be right.
You can either download "annotations_new.tar.gz" and cover the original annotation files or download the dataset again.
Please feel free to let me know if there are further issues.
